Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -75,35 +89,36 @@ def audit
       files = ARGV.resolved_formulae.map(&:path)
     end
 
-    only_cops = ARGV.value("only-cops").to_s.split(",")
-    except_cops = ARGV.value("except-cops").to_s.split(",")
+    only_cops = args.only_cops
```

Comment:
Could maybe avoid setting these variables at all and just use `args.only_cops`?

Reply:
`only_cops` and `expect_cops` are passed as arguments to `check_style_json` call below. We can't avoid setting them.